content/concepts/payment-system-basics/accounts/reserves.md

```diff
@@ -11,51 +11,58 @@ top_nav_grouping: Popular Pages
+To have an account, an address must hold a minimum amount of XRP in the shared global ledger. To fund a new address, you must receive enough XRP at that address to meet the reserve requirement. You cannot send the reserved XRP to others, but you can recover some of the XRP by [deleting the account](accounts.html#deletion-of-accounts).
 
-The reserve requirement changes from time to time due to the [Fee Voting](fee-voting.html) process, where validators can agree to new reserve settings. On [Mainnet](parallel-networks.html), the current minimum reserve requirement is **10 XRP**. (This is the cost of an address that owns no other objects in the ledger.) Each new [account](accounts.html) must set aside this much XRP. Some of this XRP can be recovered by [deleting the account](accounts.html#deletion-of-accounts).
-
-To determine the current minimum reserve requirement, use the [`server_info` command](server_info.html) and take `validated_ledger.reserve_base_xrp`.
+The reserve requirement changes from time to time due to the [Fee Voting](fee-voting.html) process, where validators can agree to new reserve settings.
 
 ## Base Reserve and Owner Reserve
 
 The reserve requirement has two parts:
 
 * The **Base Reserve** is a minimum amount of XRP that is required for each address in the ledger.
-* The **Owner Reserve** is an increase to the reserve requirement for each object that the address owns in the ledger.
+* The **Owner Reserve** is an increase to the reserve requirement for each object that the address owns in the ledger. The cost per item is also called the _incremental reserve_.
+
+The current reserve requirements on Mainnet are:
+
+- Base reserve: **10 XRP**
+- Owner reserve: **2 XRP** per item
+
+Reserves on other networks may vary.
 
+## Owner Reserves
 
-### Owner Reserves
+Many objects in the ledger (ledger entries) are owned by a particular account. Usually, the owner is the account that created the object. Each object increases the owner's total reserve requirement by the owner reserve. When objects are removed from the ledger, they no longer count against the reserve requirement.
 
-Many objects in the ledger are owned by a particular address, and count toward the reserve requirement of that address. Each object increases the reserve requirement by the Owner Reserve. On Mainnet, this is currently 2 XRP (`2000000` [drops](xrp.html#xrp-properties)) per item.
+Objects that count towards their owner's reserve requirement include: [Checks](checks.html), [Deposit Preauthorizations](depositauth.html#preauthorization), [Escrows](escrow.html), [NFT Offers](non-fungible-token-transfers.html), [NFT Pages](non-fungible-tokens.html), [Offers](offer.html), [Payment Channels](payment-channels.html), [Signer Lists](multi-signing.html), [Tickets](tickets.html), and [Trust Lines](trust-lines-and-issuing.html).
 
-When objects are removed from the ledger, they no longer count against their owner's reserve requirement.
+Some special cases:
```

Comment:
It might be easier to include each special case as a note in the corresponding bullet point above.

Reply:
That's more or less how it was before this change, and it was an overwhelming amount of info and hard to identify which ones are special cases and which aren't. My goal here is to make the full list more compact (so you can scroll over it more easily) and then only have bullet points for the exceptions.